tormenta-avro/src/main/scala/com/twitter/tormenta/scheme/avro/AvroScheme.scala

```diff
@@ -19,15 +19,20 @@ package com.twitter.tormenta.scheme.avro
 import com.twitter.bijection.Injection
 import scala.util.{ Failure, Success }
 import com.twitter.tormenta.scheme.Scheme
+import java.nio.ByteBuffer
 
 /**
  * @author Mansur Ashraf
  * @since 9/14/13
  */
 trait AvroScheme[T] extends Scheme[T] {
+  def decodeRecord(buffer: ByteBuffer)(implicit inj: Injection[T, Array[Byte]]): TraversableOnce[T] = {
```

Comment:
This part (`ByteBuffer`) is kinda concern me. I'm not sure what is a best way to mirror `ByteBuffer` in tormenta:
- we can make `ByteBuffer` -> `ByteArray` transformation in tormenta's spout/scheme itself and expose old API with `ByteArray`. 
- we can expose `ByteBuffer` itself like in this PR. But then we need to handle this mutability somehow.
What do you think?

For this avro thing it makes sense either to change passed injection to `ByteBuffer` one or remove this class completely because I can't see why we need this (I mean it doesn't have anything avro specific, right? It's just `Scheme` creation based on injection).